src/sections/follow-project/index.jsx

```diff
@@ -0,0 +1,125 @@
+import React from "react";
+import classNames from "classnames";
+import Typography from "@/components/typography";
+
+import facebook from "@/assets/facebook.svg";
+import twitter from "@/assets/twitter.svg";
+import instagram from "@/assets/instagram.svg";
+import linkedin from "@/assets/linkedin.svg";
+
+import styles from "./index.module.css";
+
+const FollowProject = () => {
+  return (
+    <div className={styles.root}>
+      <div className={styles.container}>
+        <div className={styles.title}>
+          <Typography variant="h2" weight="bold" color="purple">
+            Segue o projeto Covidografia
+          </Typography>
+        </div>
+
+        <div className={styles.socialNetworks}>
+          <div>
+            <a
+              href="#facebook"
```

Comment:
You can add `https://www.facebook.com/covidografia/`

Reply:
https://www.linkedin.com/showcase/covidografia/

Reply:
Done!